tests/Unit/ParallelAlgorithms/Events/Test_ObserveFields.cpp

```diff
@@ -504,9 +500,14 @@ SPECTRE_TEST_CASE("Unit.Evolution.dG.ObserveFields", "[Unit][Evolution]") {
                                      Spectral::Quadrature::CellCentered};
   const Mesh<2> different_all_mesh_2{8, Spectral::Basis::FiniteDifference,
                                      Spectral::Quadrature::CellCentered};
-  test_system<ScalarSystem, 1>(different_all_mesh_str, different_all_mesh_1);
-  test_system<ComplicatedSystem, 2>(different_all_mesh_str,
-                                    different_all_mesh_2);
+  test_system<ScalarSystem>(different_all_mesh_str, different_all_mesh_1);
+  test_system<ComplicatedSystem>(different_all_mesh_str, different_all_mesh_2);
+
+  // test nonuniform mesh, these cannot be parsed yet
+  const Mesh<2> nonuniform_mesh(
+      {3, 9}, {Spectral::Basis::Legendre, Spectral::Basis::Chebyshev},
+      {Spectral::Quadrature::Gauss, Spectral::Quadrature::GaussLobatto});
+  test_system<ComplicatedSystem>("", nonuniform_mesh);
```

Comment:
I think just call `test_observe` directly here instead of adding the case for the empty string.